src/views/Form/index.tsx

```diff
@@ -82,157 +94,172 @@ const Form = () => {
+      return saveUser(firstName, lastName);
     }
+    return false;
   };
 
-  const enterKeyHandler = (event: React.KeyboardEvent<HTMLInputElement>): void => {
-    if (event.key === 'Enter') {
-      handleSubmit();
-    }
-  };
-
-  const navigate = useNavigate();
-  const goBack = () => {
-    navigate('/');
-  };
-
-  const deleteByIndex = (idx: number): void => {
+  const deleteByIndex = async (idx: number): Promise<boolean> => {
     const userToDelete = users[idx];
-    deleteUser(userToDelete.firstName, userToDelete.lastName);
+    return deleteUser(userToDelete.firstName, userToDelete.lastName);
   };
 
   React.useEffect(() => {
     fetchData();
   }, []);
 
   return (
-    <Grid container spacing={4}>
-      <Grid item xs={12}>
-        <h1>NR Front End Form</h1>
-        <p>
-          This is a very simple form. Please, fill your first and last name then hit Submit!
-        </p>
-      </Grid>
-      {
-        showError && (
-        <Grid item xs={12}>
-          <Alert icon={<MdCancel size={32} />} type="error" styling="bcgov-error-background" element={errorMessage} />
-        </Grid>
-        )
-      }
-      {
-        loading && (
-          <Grid item xs={12}>
-            <CardLoader />
-          </Grid>
-        )
-      }
-      <Grid item xs={12} sm={6}>
-        <Label labelStr="First name" forStr="first-name" />
-        <TextInput
-          id="first-name"
-          isValid={firstIsValid}
-          feedback={firstFeed}
-          onChangeHandler={handleFirstName}
-          inputValue={firstName}
-          onKeyDownHandler={enterKeyHandler}
-          disabled={loading}
-        />
-      </Grid>
-      <Grid item xs={12} sm={6}>
-        <Label labelStr="Last name" forStr="last-name" />
-        <TextInput
-          id="last-name"
-          isValid={lastIsValid}
-          feedback={lastFeed}
-          onChangeHandler={handleLastName}
-          inputValue={lastName}
-          onKeyDownHandler={enterKeyHandler}
-          disabled={loading}
-        />
-      </Grid>
-      <Grid item xs={12}>
-        <Button
-          onClick={handleSubmit}
-          label="Submit"
-          styling="bcgov-normal-blue btn buttonMargin"
-          disabled={loading}
-        />
-        <Button
-          onClick={resetForm}
-          label="Reset"
-          styling="bcgov-normal-white btn buttonMargin"
-          disabled={loading}
-        />
-        <Button
-          onClick={goBack}
-          label="Back to home"
-          styling="bcgov-normal-white btn buttonMargin"
-        />
-      </Grid>
-      <Grid item xs={12}>
-        <UserTable elements={users} deleteFn={deleteByIndex} />
-      </Grid>
-    </Grid>
+    <FlexGrid>
+      <Stack gap={6}>
+        <Row>
+          <Column sm={4}>
+            <Stack gap={3}>
+              <h1>NR Front End Form</h1>
+              <p>
+                This is a very simple form. Please, fill your first and last name then hit Submit!
```

Comment:
Nitpick: I suggest we remove the first period.

It's kind of a fringe case. Depending on what happens, we can irritate the user: if our form works but the user cannot understand how to use it (specially because we didn't give them clear instructions), it might make them mad; if our form doesn't work, we'll look bad because we couldn't implement a "very simple form".

Maybe we can say its a "test form"?